addons/sale_timesheet/models/project_update.py

```diff
@@ -14,7 +14,7 @@ def _get_template_values(self, project):
         template_values = super(ProjectUpdate, self)._get_template_values(project)
         services = self._get_services_values(project)
         profitability = self._get_profitability_values(project)
-        show_profitability = bool(profitability and profitability.get('analytic_account_id') and (profitability.get('costs') or profitability.get('revenues')))
+        show_profitability = bool(self.user_has_groups('analytic.group_analytic_accounting') and profitability and profitability.get('analytic_account_id') and (profitability.get('costs') or profitability.get('revenues')))
```

Comment:
It would be better to do this  in _get_profitability_values method instead of here.

     @api.model
         def _get_profitability_values(self, project):
      
            if not self.user_has_groups('analytic.group_analytic_accounting') and costs_revenues:
            return {}
...
```suggestion
        show_profitability = bool(profitability and profitability.get('analytic_account_id') and (profitability.get('costs') or profitability.get('revenues'))
```

Reply:
@ppr-odoo yes you are right
I will make the required changes 
Thank you !